src/app/pages/MostWatchedPage/index.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, act } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { matchSnapshotAsync } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import MostWatchedPage from './MostWatchedPage';
+import pidginMostWatchedData from '#data/pidgin/mostWatched';
+import * as analyticsUtils from '#lib/analyticsUtils';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+
+fetch.mockResponse(JSON.stringify(pidginMostWatchedData));
+
+analyticsUtils.getAtUserId = jest.fn();
```

Comment:
what does this do?

Reply:
It helps with analytics  mocking but is not necessary. Taken out